ex.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.4.0 <0.9.0;
+
+contract WARP {
+  uint[] x = y;
+  uint[3] y;
+}
+
```

Comment:
Is this contract used anywhere?

Reply:
Nope, you're right it should have been removed